src/js/translations/en-US.json

```diff
@@ -68,8 +68,8 @@
   "SERVICE_ACTIONS.SDK_SERVICE_DELETE_PART_1": "You must delete",
   "SERVICE_ACTIONS.SDK_SERVICE_DELETE_PART_2": "via the DC/OS CLI. Copy and paste this command into the CLI. See the",
   "SERVICE_ACTIONS.SDK_SERVICE_DELETE_PART_3": "To clean up resources reserved by your service, run the",
-  "SERVICE_ACTIONS.SDK_SERVICE_UPDATE_PART_1": "Update your service's target configuration. Include an `options.json` file with your changes.",
-  "SERVICE_ACTIONS.SDK_SERVICE_UPDATE_PART_2": "on how to create an 'options.json' file.",
+  "SERVICE_ACTIONS.SDK_SERVICE_UPDATE_PART_1": "To Update a framework's configuration, please update",
```

Comment:
Please change to - `To update this service's configuration, please update` instead of calling out `framework`. Thats a term we don't use in the UI for end users.